app/src/pages/inside/membersPage/membersPageToolbar/memberSearchInput/memberSearchInput.jsx

```diff
@@ -0,0 +1,34 @@
+import { Fragment } from 'react';
+import Parser from 'html-react-parser';
+import { injectIntl, defineMessages } from 'react-intl';
+import classNames from 'classnames/bind';
+import { Input } from 'components/inputs/input';
+import SearchIcon from './img/ic-search-inline.svg';
+import styles from './memberSearchInput.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  searchInputPlaceholder: { id: 'MembersPage.searchByName', defaultMessage: 'Search by name' },
+});
+
+export const MemberSearchInput = injectIntl(({ intl, ...rest }) => (
+  <div className={cx('search-input')}>
+    <SearchInputWithIcon
```

Comment:
Please rebase to the fresh `develop` branch and use `InputSearch` component from there.